samples/Ch10_expressing_kernels/fig_10_9_kernel_lambda_build_options.cpp

```diff
@@ -25,22 +25,17 @@ int main() {
               << Q.get_device().get_info<info::device::name>() << "\n";
 
 // BEGIN CODE SNIP
-    // This compiles the kernel named by the specified template
-    // parameter using the "fast relaxed math" build option.
-    program p(Q.get_context());
-
-    p.build_with_kernel_type<class Add>("-cl-fast-relaxed-math");
+    kernel_id AddID = get_kernel_id<class Add>();
+    auto KernelBundle =
+        get_kernel_bundle<bundle_state::executable>(Q.get_context(), {AddID});
+    kernel Kernel = KernelBundle.get_kernel(AddID);
```

Comment:
> is it fair to say this is an equivalent sample? 

I don't quite follow what this would be the equivalent of.  It showcases kernel bundles, which are in many ways a replacement for the program class.